src/test/ValidatorKeysTool_test.cpp

```diff
@@ -59,26 +59,25 @@ class ValidatorKeysTool_test : public beast::unit_test::suite
     }
 
     void
-    testSignManifest ()
+    testCreateToken ()
     {
-        testcase ("Sign Manifest");
+        testcase ("Create Token");
 
         using namespace boost::filesystem;
 
         std::string const subdir = "test_key_file";
```

Comment:
Before this point you could test creating a token when the `validator-keys.json` file is missing.